Automattic/themes
# Independent Publisher 2: Make color annotations button rule more specific

`button:not('.menu-toggle')` was meant to avoid applying color annotation styles to the Menu toggle on mobile devices, but it has the unpleasant side effect of targeting all buttons that are not `.menu-toggle`, even if you're in the editor.

Removing the `:not('.menu-toggle')` specificity seems to work fine in the color annotations without changing things for existing users, and should fix the editor.

**Before**

<img width="1680" alt="Screen Shot 2021-08-03 at 3 29 27 PM" src="https://user-images.githubusercontent.com/2124984/128076110-2f9f08f6-79bc-4471-9551-02cb95b14ece.png">

**After**

<img width="1680" alt="Screen Shot 2021-08-03 at 3 41 22 PM" src="https://user-images.githubusercontent.com/2124984/128076152-a320d999-ab6f-45ef-acbc-5d70b4946279.png">

Testing Instructions

* Apply this PR, then find the Independent Publisher 2 theme folder locally
* Rename the theme directory and change the name in `style.css` to `Independent Publisher Test` and ZIP it.
* Switch to an Atomic site and upload the zipped theme, and activate it
* Go to the Customizer and set the color scheme on your Atomic site such that the third color circle is black (or any color you'll notice)
* Save your changes
* Go to the editor by opening a page or post
* The editor buttons should all appear normal, without a black background.

Fixes #4330

This is the same PR as D65100-code -- they should be merged at the same time.
